src/libs/actions/Report.js

```diff
@@ -1073,27 +1054,18 @@ function subscribeToNewActionEvent(reportID, callback) {
 }
 
 /**
+ * Sends callbacks and notifications for new report actions.
```

Comment:
nab, I tend to think of callbacks as functions that are called and not things that are "sent" somewhere

so I'd change this to:

>Calls callbacks and presents notifications for new report actions arriving in realtime